src/api/app/models/event/cleared_decision.rb

```diff
@@ -3,7 +3,12 @@ class ClearedDecision < Base
     receiver_roles :reporter
     self.description = 'Reported content has been cleared'
 
-    payload_keys :id, :reason, :moderator_id, :reportable_type
+    payload_keys :id, :reason, :moderator_id, :report_last_id, :reportable_type
+
+    def subject
+      decision = Decision.find(payload['id'])
+      "Cleared #{decision.reports.first.reportable&.class&.name} Report".squish
```

Comment:
@eduardoj just wondering why the `squish` is required?

Reply:
@krauselukas just for the case the interpolation of the variable returns `nil`: instead of returning `'Cleared  Report'` (with two spaces in between), only one space is returned: `'Cleared Report'`.

The same code can be found here: https://github.com/openSUSE/open-build-service/blob/91debd1dcbc8ea8be3c964d5ab2acd7ba549933b/src/api/app/components/notification_notifiable_link_component.rb#L54-L63

I would leave as it is.